src/Uno.UI/UI/Xaml/Controls/ItemsWrapGrid/ItemsWrapGrid.cs

```diff
@@ -31,7 +41,7 @@ VirtualizingPanelLayout IVirtualizingPanel.GetLayouter()
 		{
 			if (_layout == null)
 			{
-				_layout = new ItemsWrapGridLayout();
+				_layout = new ItemsWrapGridLayout();				
```

Comment:
This is likely going to fail in CI after Youssef's analyzers :)

Reply:
:D did not see this! Fixing it 🏃‍♂️🏃‍♂️

Reply:
Fixed 😅